### DISCUSSION TITLE
Supporting other databases

### DISCUSSION THREAD

**Discussion by permanar** [keystonejs/keystone#6998] [Feature requests]
Hi everyone. 

I'm currently reading the docs about the DB configuration and it only mentioned about PostgreSQL and SQLite only.

I know now Keystone 6 is using Prisma but, is there going to be any other database? 
Maybe such like MongoDB or MySQL? (AFAIK, it previously support Mongo but, I don't really know if the latest one do)

Thank you very much guys. I really appreciate this awesome work!

**Comment by Tangcuyu** [comment]
Hi，

I have same question.:)

Thank you very much.

**Comment by bladey** [comment]
Hi @permanar and @Tangcuyu,

You're right in that PostgresSQL and SQLite are the supported databases at the moment.

At the moment our next focus on database support is MongoDB, which Prisma has just finished adding the necessary parts to it to enable us to use it as we're tracking here - https://github.com/keystonejs/keystone/pull/6102 

We don’t have solid plans for MySQL at this point, but if we were to add support for another DB, it would probably be MySQL.

Thanks!

**Reply by Tangcuyu** [reply]
Thanks a lot for your answer!

**Reply by permanar** [reply]
Thanks. 
Can't wait for the future updates!

**Comment by ancs21** [comment]
Hi,

I hope Keystone supports MySQL because Prisma has it, and everyone using serverless MySQL as PlanetScale for many projects in the future.

Thanks.

**Reply by permanar** [reply]
Very likely about to support it IMO.
Just wait for the official releases I guess.

**Comment by chrisschaub** [comment]
Linking to their epic here: https://github.com/prisma/prisma/issues/8788 just for ref.

**Comment by dcousens** [comment]
I have made a specific feature requests for each database,  [Mongodb](https://github.com/keystonejs/keystone/discussions/7511) and [MySQL](https://github.com/keystonejs/keystone/discussions/7510)

**Reply by dcousens** [reply]
Locking thread in favour of these discussions (or other _specific_ discussions)
